### PR TITLE
Disruptions - Multiple disruptions on a single stop

### PR DESCRIPTION
**Pull request by alainajaffer**
**Multiple disruptions on a single stop**

Problem: When visualising disruptions when there is more than one on a single stop, currently the icons overlap each other

Solution: 

List multiple disruptions in a list form

Slightly move some of the disruptions so they're slightly apart 

“Spiderify” icons so we display one circle on the map, when clicked all disruption icons “spider” out from that circle (see link for examples of this: [https://bewithjonam.github.io/mapboxgl-spiderifier/docs/index.html) - number 2](https://github.com/nazka/map-gl-js-spiderfy?tab=readme-ov-file)

**Testing Instructions**
Navigate to 
image

Zoom in until relatively close just enough for the cluster to not become a singular icon then click on it and watch it spiderfy and click on the icons to see if pop ups show

### CODE REVIEW

File: transit_odp/frontend/assets/js/organisation-detail-map.js
Comment (DeannaSharma) [inline transit_odp/frontend/assets/js/organisation-detail-map.js:1]: Noticed a bug - for email org I made a disruption called "test 2" - it has one service type consequence which has bus service 1 and 7. On the map only bus service number 7 is showing an icon - bus service number one is missing.

Comment (DeannaSharma) [inline transit_odp/frontend/assets/js/organisation-detail-map.js:1]: Noticed a bug - if you click on the map pop up to go through to the disruption detail page. The map zooms in but no icon is displayed.

Comment (alainajaffer) [reply transit_odp/frontend/assets/js/organisation-detail-map.js:1]: fixed

File: transit_odp/frontend/assets/js/organisation-detail-map.js [hunk 1]
```diff
@@ -92,8 +137,13 @@ const httpGetAsync = (url, callback) => {
...
+  const url = disruptionId
+    ? apiRoot +
+    "disruption_detail_map_data/?orgId=" +
+    orgId.toString() +
+    "&disruptionId=" +
+    disruptionId.toString()
+    : apiRoot + "organisation_map_data/?orgId=" + orgId.toString();
 
   // Initialise Map
   mapboxgl.accessToken = mapboxKey;
```
[the remark this answers is not in the document]
Comment (alainajaffer) [reply transit_odp/frontend/assets/js/organisation-detail-map.js:149]: done

**Comment by DeannaSharma** [comment]
Reviewed by disruptions team. Awaiting review and approval from a member of the BODS team.

**Review by amitkpmguk — APPROVED** [review]
Looks good to me 👍

**Review by DeannaSharma — APPROVED** [review]

**Review by DeannaSharma — DISMISSED** [review]

**Review by DeannaSharma — APPROVED** [review]

**Review by amitkpmguk — APPROVED** [review]
Looks good to me 👍

**Pull request merged**